doc/man5/fips_config.pod

```diff
@@ -84,6 +76,130 @@ It is written-to at the same time as B<install-status> is updated.
 
 =back
 
+=head2 FIPS indicator options
+
+The following FIPS configuration option indicates if run-time checks related to
```

Comment:
Should _option_ be _options_ and the rest of the sentence updated to match the plural?